payments/tests/factories.py

```diff
@@ -194,8 +194,8 @@ class AbstractOfferFactory(factory.django.DjangoModelFactory):
 class BerthSwitchOfferFactory(AbstractOfferFactory):
     lease = factory.SubFactory(
         BerthLeaseFactory,
-        customer=factory.SelfAttribute("..customer"),
         status=LeaseStatus.PAID,
+        customer=factory.SelfAttribute("..customer"),
```

Comment:
Does this make any difference?

Reply:
Fixed. That's right, the ordering doesn't matter. I think I had made the change when I started the branch and then afterwards the same change was introduced to the main branch from another commit.